docs/source/getting_started/overview.rst

```diff
@@ -55,6 +55,15 @@ JupyterLab is served from the same `server
 `notebook document format <https://nbformat.readthedocs.io/en/latest/>`__ as the
 classic Jupyter Notebook.
 
+.. toctree::
```

Comment:
Could we add this one or at least use only one level of titles? The list of changelog is really long and make the last paragraph about releases hidden as user will most likely give up scrolling the version list.

Reply:
Latest commit moves `changelog` to its own toctree just after this one, with a maxdepth of 1, so it's more readable now

Reply:
Thanks for the update. Could it be that the commit was not pushed?